src/components/PullRequestList/PullRequestListItem/index.js

```diff
@@ -1,4 +1,4 @@
-/* @flow */
+// TODO: fix flow for build property
 
```

Comment:
Is it a lot of work to model a build? This really seems like a step in the wrong direction...